x-pack/legacy/plugins/file_upload/index.js

```diff
@@ -23,11 +21,27 @@ export const fileUpload = kibana => {
     },
 
     init(server) {
-      const { xpack_main: xpackMainPlugin } = server.plugins;
+      const coreSetup = server.newPlatform.setup.core;
+      const pluginsSetup = {};
 
-      mirrorPluginStatus(xpackMainPlugin, this);
-      fileUploadRoutes(server);
-      makeUsageCollector(server);
+      // legacy dependencies
+      const __LEGACY = {
+        route: server.route.bind(server),
+        plugins: {
+          elasticsearch: server.plugins.elasticsearch,
+        },
+        savedObjects: {
+          getSavedObjectsRepository:
+            server.savedObjects.getSavedObjectsRepository
```

Comment:
single line

Reply:
line-length is enforced with the linting, so no need to make it shorter when it does not need to be